File: Visit.java
import java.time.LocalDate;
import java.util.ArrayList;

public class Visit {
    private static int cont = 1;
    private String ID;
    private String employeeInCharge;
    private ArrayList<String> visitants = new ArrayList<String>();
    private double totalPrice;
    private String dayOfVisit;
    private int adults;
    private int children;

    public Visit(String employeeInCharge, ArrayList<String> visitants, double totalPrice, int adults, int children) {
        this.employeeInCharge = employeeInCharge;
        this.ID = String.format("%03d", cont);
        this.visitants = visitants;
        this.totalPrice = totalPrice;
        this.dayOfVisit = LocalDate.now().toString();
        this.adults = adults;
        this.children = children;
    }

    public void printVisitants(){
        System.out.println("Lista de visitantes: ");
        for (String visitant : visitants){
            System.out.println("\t"+visitant);
        }
    }

    public String getEmployeeInCharge() {
        return employeeInCharge;
    }

    public ArrayList<String> getVisitants() {
        return visitants;
    }

    public double getTotalPrice() {
        return totalPrice;
    }

    public String getDayOfVisit() {
        return dayOfVisit;
    }

    public int getAdults() {
        return adults;
    }

    public int getChildren() {
        return children;
    }

    public String getID() {
        return ID;
    }
}

File: Visit.py
class Visit:
    cont=1
    visitors=[]
    def __init__(self,employee_in_charge,visitor,total_price,adults,children):
        self.employee_in_charge= employee_in_charge
        self.visitors.append(visitor)
        self.total_price= total_price
        self.adults=adults
        self.children =children
        self.ID=str(Visit.cont).zfill(3)
        Visit.cont+=1

    def visits_info(self):
        print("ID de la visita: ", self.ID)
        print("Empleado a cargo: ",self.employee_in_charge)
        print("Visitantes: ",self.visitors)
        print("Precio total de la visita: ", self.total_price)
        print("Adultos en la visita: ", self.adults)
        print("Niños en la visita: ",self.children)

    def print_visitors(self):
        print("Lista de visitantes:")
        for visitor in self.visitors:
            print(visitor)

    def visit_id(self):
        print("Id de la visita: ", self.ID )
    